src/sentry/coreapi.py

```diff
@@ -174,16 +174,16 @@ def __init__(self, agent=None, version=None, project_id=None,
         self.log = ClientLogHelper(self.context)
 
     def auth_from_request(self, request):
-        if request.META.get('HTTP_X_SENTRY_AUTH', '').startswith('Sentry'):
+        if request.META.get('HTTP_X_SENTRY_AUTH', '')[:7] == 'Sentry ':
```

Comment:
Ok, according to [rfc2617](https://www.ietf.org/rfc/rfc2617.txt) it does state the auth-scheme is case-insensitive.